### ISSUE TITLE
pip install results in GEOS mismatch

### ISSUE THREAD

**Issue by nreinicke** [NREL/mappymatch#18]
When installing yamm via pip, there ends up being a mismatch between the shapely GEOS version and the pygeos version (see below). This doesn't seem to happen when installing via conda-forge. 

```
../yamm/lib/python3.8/site-packages/geopandas/_compat.py:111: UserWarning: The Shapely GEOS version (3.10.2-CAPI-1.16.0) is incompatible with the GEOS version PyGEOS was compiled with (3.10.1-CAPI-1.16.0). Conversions between both will be slow.
```

**Comment by uranusjr** [comment]
I believe this is a known issue between Shapely and PyGEOS, and they are trying to resolve the issue by merging the projects (Shapely 2.0) and synchronise the PyPI releases. But there’s probably not much we can do here until that happens. (Note that code should work, but likely not as efficient as everyone would like.)

**Comment by LiamMeron** [comment]
This is closed by issue #71 as we will remove PyGeos as direct dependency to rely upon Shapely exclusively
